fission/package.go

```diff
@@ -387,3 +387,30 @@ func pkgDelete(c *cli.Context) error {
 
 	return nil
 }
+
+func pkgRetry(c *cli.Context) error {
+	client := getClient(c.GlobalString("server"))
+
+	pkgName := c.String("name")
+	if len(pkgName) == 0 {
+		log.Fatal("Need name of package, use --name")
+	}
+	pkgNamespace := c.String("pkgNamespace")
+
+	pkg, err := client.PackageGet(&metav1.ObjectMeta{
+		Name:      pkgName,
+		Namespace: pkgNamespace,
+	})
+	checkErr(err, "find package")
+
+	if pkg.Status.BuildStatus != fission.BuildStatusFailed {
+		log.Fatal(fmt.Sprintf("Package %v is not in %v state.",
+			pkg.Metadata.Name, fission.BuildStatusFailed))
+	}
+
+	updatePackage(client, pkg, "", "", "", "", "", true)
+
+	fmt.Printf("Set package '%v' to %v state\n", pkg.Metadata.Name, fission.BuildStatusPending)
```

Comment:
How about simply "Retrying build for pkg __.  Use `fission pkg info --name ___` to view status."